src/routes/DigitalAssets/routes/Send/modules/digitalAssetsSend.js

```diff
@@ -9,6 +9,9 @@ export const GO_TO_NEXT_STEP = '@@digitalAssetsSend/GO_TO_NEXT_STEP'
 export const GO_TO_PREV_STEP = '@@digitalAssetsSend/GO_TO_PREV_STEP'
 export const SET_CURRENT_STEP = '@@digitalAssetsSend/SET_CURRENT_STEP'
 
+export const SET_GAS_PRICE_VALUE = '@@digitalAssetsSend/SET_GAS_PRICE_VALUE'
+export const SET_GAS_LIMIT_VALUE = '@@digitalAssetsSend/SET_GAS_LIMIT_VALUE'
+
```

Comment:
`SET_GAS_PRICE_VALUE_TO_DEFAULT`?

Reply:
no, these values should be setted, and then never changed, because they are needed to compare with user values.
Currently gas management does not work.

Reply:
This values will be changed, when
- another digital asset selected
- priority changed
- on 1st step confirm, we are re-requesting gas price, it can be changed (and you can check and confirm it)

Reply:
only first case, and we will get DEFAULT values for another asset